# Atlas.Y Dataset

> Atlas.Y Dataset is licensed under the [Attribution-NonCommercial 4.0 International (CC BY-NC 4.0) license](https://creativecommons.org/licenses/by-nc/4.0). You are free to share and adapt the dataset for non-commercial purposes, as long as proper attribution is given. For commercial use, please [contact us](mailto:tongji_china2019@163.com) to request permission.

English | [简体中文](README_zh.md)

![](assets/Logo.png)

## Signal Peptide Dataset

The Signal Peptide Dataset is designed to facilitate research on protein subcellular localization and transport. Signal peptides are short sequences located at the N-terminus of newly synthesized proteins, responsible for determining the subcellular localization of these proteins. The data in this database is sourced from the dataset used in the development of the DeepLoc 2.1 deep learning model by Marius Thrane Ødum et al. To align with the goals of our project, we specifically filtered the dataset to include only proteins from eukaryotic organisms, extracted the signal peptides, categorized them, and assigned unique identifiers to enable efficient querying. This dataset is particularly suitable for bioinformatics research, protein design, and cell biology experiments focused on predicting subcellular localization.

[Signal_Peptide.csv](Signal_Peptide.csv)

## Linker Dataset

This Linker Dataset is designed to study the linkers between signal peptides and target proteins, assisting in the design and optimization of fusion proteins, particularly in research related to protein subcellular localization and transport. The dataset is divided into two tables: the Classical Linker Data Table and the Natural Linker Data Table.

* **Classical Linker Table**: This table contains linkers that have been extensively reviewed in the literature and categorized based on their rigidity and flexibility. The linker sequences in this table are suitable for use in protein design, molecular biology, and synthetic biology engineering projects.

  [Classical_Linker.csv](Classical_Linker.csv)

* **Natural Linker Table**: This table consists of short peptides derived from natural protein sequences without artificial optimization. These natural linkers are generated by removing signal peptides and conserved regions, following the method used by the 2021 Sun Yat-sen University iGEM team. We used protein sequences from the DeepLoc 2.1 dataset, and identified conserved regions using NCBI's Conserved Domain Database (CDD) and its Batch CD-Search tool.

  [Natural_Linker.csv](Natural_Linker.csv)

This dataset has broad applications, including protein engineering, molecular design, signal peptide function research, and bioinformatics analysis. The two tables in the dataset provide scientists with a foundational resource for efficiently querying and utilizing linker sequences.

## Optogenetic Protein Pair Dataset

### Principle of the Optogenetic System

Specific light conditions can control the interactions between optogenetic protein pairs, allowing precise regulation of reactions inside cells. This system relies on two proteins: a light-sensitive protein and a receptor protein. Under light stimulation, they bind or separate, which starts or stops a specific cellular reaction.

1. **Light-sensitive Protein’s Response to Light**: Light-sensitive proteins change shape when exposed to specific wavelengths of light. They contain special light-absorbing molecules called chromophores. When these molecules absorb light, they cause the protein to change its structure, which then reveals or hides binding sites. This determines whether the light-sensitive protein can connect with its receptor protein partner.

2. **Receptor Protein’s Role**: Receptor proteins usually cannot respond to light themselves. Instead, they recognize and bind to light-sensitive proteins only after they change shape due to light. This light-induced interaction can switch on or off various cellular processes.

3. **Precise Control of Reactions**: Light-sensitive protein pairs are powerful tools for controlling where and when reactions happen in cells. Since light can be focused on a specific area and timed precisely, researchers can control protein interactions in just the right place and at the right time, minimizing any unwanted effects on other parts of the cell.

### Light-sensitive Protein (with Signal Peptide) Data Table

#### Overview

This table contains information on light-sensitive proteins that have a signal peptide. The signal peptide helps these proteins reach a specific location in the cell, such as the cell membrane or nucleus. Under certain light conditions, these proteins attract their receptor partners to bind with them, enabling precise control of cell activities.

[Dynamic_Navigator_Pro.csv](Dynamic_Navigator_Pro.csv)

#### Purpose of the Dataset

This data table is used in the **dynamic module** of **Atlas.Y**. The proteins listed here can be targeted to specific locations in the cell because of their signal peptides. Once localized, they can guide other proteins to the same spot when the right light is applied. Each light-sensitive protein is also linked with a fluorescent protein, making it easier to see where it is in the cell.

### Receptor Protein (without Signal Peptide) Data Table

#### Overview

This table contains information on receptor proteins that pair up with light-sensitive proteins under specific light conditions. These receptor proteins do not have a signal peptide themselves, but they move to the correct location in the cell by binding to a light-sensitive protein that does. To visualize these interactions, the receptor proteins are fused with fluorescent proteins, allowing researchers to track them inside the cell.

[Dynamic_Sensor_Pro.csv](Dynamic_Sensor_Pro.csv)

#### Purpose of the Dataset

This data table is used in the **dynamic module** of **Atlas.Y**. When a user provides a target protein, the software will connect it between the receptor protein and a fluorescent protein, creating a new fusion protein system. This setup allows users to see and control how the target protein behaves under different light conditions. Using this data, **Atlas.Y** helps users design light-controlled fusion proteins to precisely regulate protein movement and interactions inside cells.

### Data Source

The data in this table mainly come from the following studies and databases:

1. Kennedy, M., Hughes, R., Peteya, L., Schwartz, J., Ehlers, M. & Tucker, C. (2010). Rapid blue-light–mediated induction of protein interactions in living cells. Nature Methods, 7, 973–975. https://doi.org/10.1038/nmeth.1524

2. Müller, K., Engesser, R., Metzger, S., Schulz, S., Kämpf, M. M., Busacker, M., Steinberg, T., Tomakidi, P., Ehrbar, M., Nagy, F., Timmer, J., Zubriggen, M. D., & Weber, W. (2013). A red/far-red light-responsive bi-stable toggle switch to control gene expression in mammalian cells. Nucleic Acids Research, 41(7), e77. https://doi.org/10.1093/nar/gkt002

3. Kaberniuk, A., Shemetov, A., & Verkhusha, V. (2016). A bacterial phytochrome-based optogenetic system controllable with near-infrared light. Nature Methods, 13, 591–597. https://doi.org/10.1038/nmeth.3864

4. Sequences and information on sensor proteins and fluorescent proteins from the NCBI database.